kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/ext/TypeExtensions.kt

```diff
@@ -54,14 +54,13 @@ fun Type.toTypeName(): TypeName = ClassName(
 
 fun Type.isCompatibleList(): Boolean = when (fqName) {
     "godot.core.GodotArray", "godot.core.VariantArray" -> true
```

Comment:
I think "godot.core.GodotArray" is not even in the codebase anymore. There was a time when we had 2 different array classes, but it was removed a long time ago.